projects/igniteui-angular/src/lib/simple-combo/README.md

```diff
@@ -0,0 +1,317 @@
+
+## Features
+
+### Selection
+
+Combo selection depends on the `[valueKey]` input property:
+
+- If a `[valueKey]` is specified, **all** methods and events tied to the selection operate w/ the value key property of the combo's `[data]` items:
+```html
+    <igx-simple-combo [data]="myCustomData" valueKey="id" displayKey="text"></igx-simple-combo>
+```
+```typescript
+export class MyCombo {
+    ...
+    public combo: IgxSimpleComboComponent;
+    public myCustomData: { id: number, text: string } = [{ id: 0, name: "One" }, ...];
+    ...
+    public ngOnInit(): void {
+        // Selection is done only by valueKey property value
+        this.combo.select(0);
+    }
+}
+```
+
+- When **no** `valueKey` is specified, selection is handled by **equality (===)**. To select items by object reference, the `valueKey` property should be removed:
+```html
+    <igx-simple-combo [data]="myCustomData" displayKey="text"></igx-simple-combo>
+```
+```typescript
+export class MyCombo {
+    public ngOnInit(): void {
+        this.combo.select(this.data[0]);
+    }
+}
+```
+
+### Value Binding
+
+If we want to use a two-way data-binding, we could just use `ngModel` like this:
+
+```html
+<igx-simple-combo #combo [data]="data" valueKey="id" displayKey="text" [(ngModel)]="value"></igx-simple-combo>
+```
+```typescript
+export class MyExampleComponent {
+    ...
+    public data: {text: string, id: number, ... }[] = ...;
+    ...
+    public value: number = ...;
+}
+```
+
+When the `data` input is made up of complex types (i.e. objects), it is advised to bind the selected data via `valueKey` (as in the above code snippet). Specify a property that is unique for each data entry and pass in a value to the combo that is the same as the unique identifier in the data set.
+
+If you want to bind the selected data by reference, **do not** specify a `valueKey`:
+
+```html
+<igx-simple-combo #combo [data]="data" displayKey="text" [(ngModel)]="value"></igx-simple-combo>
+```
+```typescript
+export class MyExampleComponent {
+    ...
+    public data: {text: string, id: number, ... }[] = ...;
+    ...
+    public value: {text: string, id: number, ...} = this.items[0];
+}
+```
+
+<div class="divider--half"></div>
+
+### Filtering
+Unlike the `igx-combo`, filtering in the `igx-simple-combo` is always enabled.
+
+<div class="divider--half"></div>
+
+### Custom Values
+Enabling the custom values will add values that are missing from the list, using the combo's interface.
+
+```html
+<igx-simple-combo [allowCustomValues]="true"></igx-simple-combo>
+```
+
+<div class="divider--half"></div>
+
+### Disabled
+You can disable the combo using the following code:
+
+```html
+<igx-simple-combo [disabled]="true"></igx-simple-combo>
+```
+
+<div class="divider--half"></div>
+
+### Grouping
+Defining a combo's groupKey option will group the items, according to that key.
+
+```html
+<igx-simple-combo [groupKey]="'primaryKey'"></igx-simple-combo>
```

Comment:
using `primaryKey` as `groupKey` here could be missleading. Change it with `groupKey` or something else, but not `primaryKey`.